lib/matplotlib/tests/test_mathtext.py

```diff
@@ -109,7 +109,7 @@
     r'${xyz}^k{x}_{k}{x}^{p}{y}^{p-2} {d}_{i}^{j}{b}_{j}{c}_{k}{d} {x}^{j}_{i}{E}^{0}{E}^0_u$',
     r'${\int}_x^x x\oint_x^x x\int_{X}^{X}x\int_x x \int^x x \int_{x} x\int^{x}{\int}_{x} x{\int}^{x}_{x}x$',
     r'testing$^{123}$',
-    ' '.join('$\\' + p + '$' for p in sorted(mathtext.Parser._accentprefixed)),
```

Comment:
This should be changed back, but use `_mathtext.Parser._accentprefixed`. Will fix that after I get other review comments.

Reply:
I'm not following what you're doing here, but it seems you'll change this anyway. The rest looks fine.

Reply:
I removed this as `mathtext.Parser` was being removed, but then I realized (when checking what I had actually done) that this was just moved.